src/main/java/org/terasology/simpleFarming/systems/VineAuthoritySystem.java

```diff
@@ -16,13 +16,11 @@
 package org.terasology.simpleFarming.systems;
 
 import org.terasology.logic.common.ActivateEvent;
-import org.terasology.simpleFarming.components.BushDefinitionComponent;
-import org.terasology.simpleFarming.components.SeedDefinitionComponent;
-import org.terasology.simpleFarming.components.VineNodeComponent;
+import org.terasology.logic.inventory.ItemComponent;
+import org.terasology.simpleFarming.components.*;
```

Comment:
Star import, shouldn't be used

Reply:
Thanks,  I have fixed my IDE settings and un-starred these.